index.js

```diff
@@ -94,6 +100,7 @@ const getPastEvents = (startBlockNumber) => {
     .then((events) => {
       events.forEach(function (batchTranfer) {
         var {
```

Comment:
The proper way to define vars is with `let` now: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let `var` will define the variable globally...

Reply:
A find/replace should be enough to fix that I think...